README.md

```diff
@@ -1,7 +1,7 @@
 Bluefin
 -------
 
-[![CircleCI](https://circleci.com/gh/dailymuse/conflator.svg?style=svg&circle-token=698d6375f0be2d64d95d72b3302b1bc739cc08b9)](https://circleci.com/gh/dailymuse/conflator) [![codecov](https://codecov.io/gh/dailymuse/conflator/branch/develop/graph/badge.svg?token=gd7zCcWUMv)](https://codecov.io/gh/dailymuse/conflator)
+[![CircleCI](https://circleci.com/gh/dailymuse/bluefin.svg?style=svg)](https://circleci.com/gh/dailymuse/bluefin) [![codecov](https://codecov.io/gh/dailymuse/bluefin/branch/develop/graph/badge.svg?token=gd7zCcWUMv)](https://codecov.io/gh/dailymuse/bluefin)
```

Comment:
I think you'll want to remove the `token= gd7zCcWUMv` for codecov since this is a public repo